plugins/metrics/scripts/ai_docs_telemetry.py

```
@@ -0,0 +1,279 @@
+        if project_filter and project_filter not in str(session_file):
+            continue
+
+        processed_count += 1
+
+        # Quick pre-filter: check if file contains ai-docs markers
+        try:
+            content = session_file.read_text()
+            if not any(marker in content for marker in _AI_DOCS_MARKERS):
+                continue
+        except Exception:
+            continue
```

Ruff (0.15.15)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)